apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/TaskExecutor.java

```diff
@@ -104,17 +103,11 @@ public static String getTaskId(Object task) {
          * Tasks submitted to TaskExecutor get wrapped in a ScheduledFutureTask, which this branch
          * covers.
          */
-
-        Field timeField = task.getClass().getDeclaredField("time");
-        timeField.setAccessible(true);
-        return "" + timeField.get(task);
+        return task.getClass().toString();
```

Comment:
How is this unique to a particular task?  Does that not matter?  Previously a time stamp was being used to provide some uniqueness but all instances of a given class will now have the same `id`.

Reply:
Added the hashcode to the end for a bit more uniqueness as discussed in slack. Thanks for the idea!